test/infra/vsphere/packer-vsphere-airgap.yaml.tmpl

```diff
@@ -10,7 +10,7 @@ packer:
   ssh_bastion_private_key_file: /kib/vsphere-tests.pem
   cluster: "zone1"
   datacenter: "dc1"
-  datastore: "esxi-06-disk1"
+  datastore: "ovh-nfs"
```

Comment:
moving back to NFS storage to make sure we dont lose templates with vcenter upgrade/migrations etc. This will slow down the build process by 5 to 10 minutes.